modules/EvseSlac/main/slacImpl.cpp

```diff
@@ -61,7 +61,7 @@ void slacImpl::run() {
     }
 
     // setup callbacks
-    ContextCallbacks callbacks;
+    slac::fsm::evse::ContextCallbacks callbacks;
     callbacks.send_raw_slac = [&slac_io](slac::messages::HomeplugMessage& msg) { slac_io.send(msg); };
 
     callbacks.signal_dlink_ready = [this](bool value) { publish_dlink_ready(value); };
```

Comment:
This is never called. We could implement the proprietary extension to find out whether the PLC actually linked with a peer, but not sure if this is portable to other PLC chips. So for now it would be nice to signal this when we go to MATCHED and when we leave MATCHED

Reply:
The comment was about the signal_dlink_ready

Reply:
Correct, I need to review the interface